great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -106,6 +106,7 @@ def run(
             []
             if self.domain_builder is None
             else self.domain_builder.get_domains(
+                rule_name=self.name,
```

Comment:
This was added because having the same domain being referred to by different `Rule`